evil-profiler.el

```diff
@@ -55,8 +54,8 @@
     "i" 'profiler-report-describe-entry
     "=" 'profiler-report-compare-profile
 
-    ;; goto
-    "gd" 'profiler-report-find-entry
```

Comment:
Was it necessary to remove 'gd' here?

Reply:
Not per se, but I'm trying to make sense of the rationale.
Should "follow" and "go to definition" be the same concepts and bindings? If not, then this profiler command is a "follow" and should be bound to `RET` and not `gd`.

Let's discuss this in #27.